kitsune/questions/cron.py

```diff
@@ -186,9 +186,11 @@ def report_employee_answers():
     num_answered = questions.filter(num_answers__gt=0).count()
 
     # Total number of questions answered by user in tracked_group
-    num_answered_by_tracked = Answer.objects.filter(
-        question__in=questions,
-        creator__in=tracked_users).values_list('question_id').distinct().count()
+    num_answered_by_tracked = {}
+    for user in tracked_users:
+        num_answered_by_tracked[user.username] = Answer.objects.filter(
+            question__in=questions,
+            creator=user).values_list('question_id').distinct().count()
```

Comment:
Oh, you're right, this is different. I imagine it is possible, but it probably isn't worth the trouble, since this is a cronjob and not an api call.